apps/api/src/server/app/utils/address-formatter-lowercase.js

```diff
@@ -0,0 +1,11 @@
+const formatAddressLowerCase = function(address) {
+	return {
+		addressLine1: address.addressLine1,
+		addressLine2: address.addressLine2,
+		town: address.town,
+		county: address.county,
+		postCode: address.postcode
```

Comment:
FYI, "postcode" is a single word and so having this function to change its case isn't really necessary...